server/system_manager.py

```diff
@@ -34,8 +36,6 @@
 from server import active_site
 # Necessary so pickle can access class definitions from vehicle.
 sys.path.append('../vehicle')
```

Comment:
Re-ordering like this do break the code, and the formatter keeps doing. In a follow up PR I added the `/acorn/vehicle` directory to `PYTHONPATH` so the server can be up, but for scripts in `reporting/` and `bringup/` etc to work, `PYTHONPATH` needs to be defined somewhere on RPi, preferably `/etc/profile` or `/etc/profile.d/....sh`. It's unclear to me where's the best place to do that, `bringup/bringup.sh`?